azure-pipelines.yml

```diff
@@ -3,6 +3,4 @@
 # -*- mode: yaml -*-
 
 jobs:
-  - template: ./.azure-pipelines/azure-pipelines-linux.yml
-  - template: ./.azure-pipelines/azure-pipelines-win.yml
-  - template: ./.azure-pipelines/azure-pipelines-osx.yml
```

Comment:
Is this because `noarch` only needs to be built once?

Reply:
Yes.